next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
```

Comment:
do we need this?

Reply:
Yes, those types are needed by the compiler to find the framework types